tensorflow/core/kernels/dml_concat_op.cc

```diff
@@ -240,7 +240,7 @@ class DmlConcatKernel : public DmlKernel {
 
     // TFDML #24881131
     if (Is64BitSignedIntegerType(ctx->GetOutputDataType(0))) {
-      result = dml::ConvertInt32ToInt64(scope, result);
+      result = dml::ConvertInt32ToInt64(result);
```

Comment:
>ConvertInt32ToInt64 [](http://example.com/codeflow?start=20&length=19)

Given concat supports int64/uint64 directly, can we just pass int64 to dml::Join instead of the upcast later? Same applies to a few other ops in this CR.

This is a good next step either way though, if that's deferred for later.

Reply:
Agree, I'll do it incrementally.